jupyter_console/app.py

```diff
@@ -142,7 +143,7 @@ def init_banner(self):
         self.shell.show_banner()
         # Make sure there is a space below the banner.
         if self.log_level <= logging.INFO: print()
-    
+
```

Comment:
BTW, in general we ask people to configure their editor not to strip whitespace, because it inflates diffs and can lead to spurious merge conflicts. It doesn't matter much when it's only a few lines like here, but it's good to bear in mind if you contribute bigger changes. :-)